Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -442,29 +442,19 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         completion: @escaping (Result<EvaluationResult, Error>) -> Void
     ) {
         do {
-            if localFileSystem.isFile(manifestPath) {
+            try withTemporaryFile(suffix: ".swift") { tempFile, cleanupTempFile in
+                try localFileSystem.writeFileContents(tempFile.path, bytes: ByteString(manifestContents))
```

Comment:
I think you can just pass the string now

Reply:
Doesn't seem to work since we only have a `[UInt8]`.